lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -307,10 +312,19 @@ sub wait_for_messages {
 sub _on_message {
     my ($self, $redis, $raw_message) = @_;
 
-    my $message = eval { decode_json_utf8($raw_message) };
+    my $message = {};
+
+    try{
```

Comment:
Needs tidy

Reply:
There is no rule to tidy up the code, but change several lines manually.